src/parser/Parser.ts

```diff
@@ -80,9 +80,23 @@ import {
 } from './Expression';
 import { Diagnostic, Range } from 'vscode-languageserver';
 import { Logger } from '../Logger';
-import { isCallExpression, isCallfuncExpression, isClassMethodStatement, isDottedGetExpression, isIndexedGetExpression, isVariableExpression } from '../astUtils/reflection';
+import { isCallExpression, isCallfuncExpression, isClassMethodStatement, isCommentStatement, isDottedGetExpression, isIndexedGetExpression, isVariableExpression } from '../astUtils/reflection';
 import { createVisitor, WalkMode } from '../astUtils/visitors';
 
+function createReferences(): References {
```

Comment:
Can you move this `createReferences` somewhere other than the top of the file? I prefer keeping the primary class as the topmost item in a file. It could go to the bottom of the file, get set as a static method on the class, or even just a regular method since javascript prototypes should keep this function very fast and it doesn't have any use outside of this class anyway.